code/mathematical-algorithms/Check_is_square/square_checker.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func square_check(a float64) int {
```

Comment:
Make the return type bool, and call the function `is_square`. Return true if the number is a perfect square, and false otherwise.